addons/im_livechat/report/im_livechat_report_channel_views.xml

```diff
@@ -1,16 +1,17 @@
 <?xml version="1.0"?>
 <odoo>
     <data>
-
+    
         <record id="im_livechat_report_channel_view_pivot" model="ir.ui.view">
             <field name="name">im_livechat.report.channel.pivot</field>
             <field name="model">im_livechat.report.channel</field>
             <field name="arch" type="xml">
                 <pivot string="Livechat Support Statistics" disable_linking="1" sample="1">
-                    <field name="technical_name" type="row"/>
-                    <field name="duration" type="measure" string="Duration of Session (min)"/>
-                    <field name="nbr_message" type="measure" string="Messages per session"/>
-                    <field name="is_without_answer" invisible="True"/>
+                    <field name="partner_id" type="row"/>
+                    <field name="nbr_channel" type="measure"/>
+                    <field name="time_to_answer" type="measure"/>
+                    <field name="duration" type="measure"/>
+                    <field name="rating" type="measure"/>
```

Comment:
This should probably use the rating % widget, I'm not sure if it's merged yet or still in another PR.